sdk/purview/azure-analytics-purview-scanning/pom.xml

```diff
@@ -40,12 +40,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.20.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
-    </dependency>
-    <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-experimental</artifactId>
-      <version>1.0.0-beta.18</version> <!-- {x-version-update;com.azure:azure-core-experimental;dependency} -->
+      <version>1.21.0-beta.1</version> <!-- {x-version-update;beta_com.azure:azure-core;dependency} -->
     </dependency>
     <dependency>
```

Comment:
clean up unused dependencies e.g. `javax.json`

Reply:
We used javax.json in src/samples/java/com/azure/analytics/purview/scanning/ListAllSystemScanRulesets.java, I will clean up other unused ones.